docs/apigateway.md

```diff
@@ -1,53 +1,105 @@
-# API Gateway (Experimental)
+# API Gateway
 
-[Web Actions](webactions.md) are out from experimental phase.
+OpenWhisk actions can benefit from being managed by API management.
 
-Web Actions allows you to invoke an action with HTTP methods other than POST and without the action's authorization API key.
+The API Gateway acts as a proxy to [Web Actions](webactions.md) and provides them with additional features features including HTTP method routing, security and rate limiting policies and more.
```

Comment:
HTTP method routing => routing to an action based on HTTP method. Not sure how you'd put in more explanation without cluttering the document though?